src/styles/filter.css

```diff
@@ -0,0 +1,28 @@
+.filter-nav {
+  display: flex;
+  width: 600px;
+  margin: 0rem auto;
+  padding: 1rem;
+  background-color: rgb(240, 235, 235);
+}
+input {
+  margin-left: 1rem;
+}
+
+@media only screen and (max-width: 650px) {
+  .filter-nav {
+    width: max-content;
+    margin: 1rem auto;
+  }
+}
+@media only screen and (max-width: 420px) {
+  .filter-nav {
+    width: max-content;
+    margin: 1rem auto;

```

Comment:
imo this wont be needed coz u already have same style for upto 650px

Reply:
Right, I forgot to compare.